tests/test_env.py

```diff
@@ -74,47 +75,6 @@ def test_venv_pyton_env_empty_string(tmpdir):
     assert e.value.args[0] == "The env_path cannot be an empty string."
 
 
-@pytest.mark.slowtest
-def test_basic_install(tmpdir):
```

Comment:
Why did this one have to go? Non PEP508 compliant?

Reply:
added back